docs/products/metrics/concepts/metrics-overview.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Aiven for Metrics® overview
+---
+
+Aiven for Metrics®, powered by Thanos, simplifies managing and analyzing large volumes of metrics data for businesses of all sizes. It offers a scalable, reliable, and efficient service suitable for businesses of all sizes. This service simplifies the management of large-scale metrics systems, allowing organizations to focus on deriving insights from their data.
+
+## Key components
+
+Aiven for Metrics incorporates several core Thanos components:
+
+- **Thanos Metrics Query**: Allows users to query and visualize metrics from real-time
+  and historical data sources, aggregating data from various sources for a unified view.
```

Comment:
Thanos Query Frontend is probably also worth mentioning. https://thanos.io/tip/components/query-frontend.md/

Reply:
Added this to the list.